prov/rxm/src/rxm_ep.c

```diff
@@ -1019,8 +1027,10 @@ rxm_ep_sar_tx_send(struct rxm_ep *rxm_ep, struct rxm_conn *rxm_conn, void *conte
 			dlist_insert_tail(&tx_buf->hdr.entry,
 					  &tx_entry->deferred_tx_buf_list);
 		}
-		segs_cnt--;
+
+		tx_entry->segs_left++;
```

Comment:
Is this being used elsewhere? Also why is `segs_left` incremented after every send?